### PR TITLE
Add text

### PR DESCRIPTION
**Pull request by Danielle-P**

### CODE REVIEW

File: src/reset.less [hunk 1]
```diff
@@ -85,6 +85,10 @@ a {
   }
 }
 
+a[href='mailto:'] {
```
Comment (benadamstyles) [inline src/reset.less]:
```suggestion
a[href^='mailto:'] {
```

File: src/the-day.html [hunk 2]
```diff
@@ -31,24 +31,38 @@ <h1>On the Day</h1>
   <div class="content">
     <section>
       <h2>Timings</h2>
+      <div class="sub-section">
+        <p>We can't wait to celebrate with you!</p>
```
Comment (benadamstyles) [inline src/the-day.html]:
```suggestion
        <p>We can&rsquo;t wait to celebrate with you!</p>
```

File: src/the-day.html [hunk 3]
```diff
@@ -31,24 +31,38 @@ <h1>On the Day</h1>
...
         <h3>3<span class="superscript">pm</span></h3>
-        <p>Something is happening</p>
+        <p>The ceremony followed by bubbles</p>
       </div>
       <hr />
       <div class="sub-section">
-        <h3>5<span class="superscript">pm</span></h3>
-        <p>Something is happening</p>
+        <h3>4:30<span class="superscript">pm</span></h3>
+        <p>Food by Harewood Food & Drink Project</p>
```
Comment (benadamstyles) [inline src/the-day.html]: Add a link

File: src/travel-accommodation.html [hunk 4]
```diff
@@ -52,34 +52,41 @@ <h4>
 
         <div class="sub-section">
           <h3>By Train</h3>
-          <p>Any information on travelling by train/directions from station</p>
+          <p>The Brewery is really easy to get to from Leeds train station. From there it's either a 10-15 minute walk
```
Comment (benadamstyles) [inline src/travel-accommodation.html]:
```suggestion
          <p>The Brewery is really easy to get to from Leeds train station. From there it&rsquo;s either a 10-15 minute walk
```

File: src/travel-accommodation.html [hunk 5]
```diff
@@ -52,34 +52,41 @@ <h4>
...
-            <h4>Other Taxi</h4>
-            <a href="tel:" class="phone-number">0113 123 4567</a>
+            <h4>Uber</h4>
+            <a href="https://www.uber.com/gb/en/ride/" target="_blank" rel="noopener">uber.com</a>
           </div>
         </div>
 
         <div class="sub-section">
           <h3>Parking</h3>
+          <p>Unfortunately, there isn't any parking at the Brewery itself. But the good news is there's a fair bit of
```
Comment (benadamstyles) [inline src/travel-accommodation.html]:
```suggestion
          <p>Unfortunately, there isn&rsquo;t any parking at the Brewery itself. But the good news is there&rsquo;s a fair bit of
```

File: src/travel-accommodation.html [hunk 6]
```diff
@@ -88,22 +95,29 @@ <h4>Car Park</h4>
 
     <hr />
 
-    <section>
+    <section class="hotels">
       <h2>Accommodation</h2>
+      <p>We have some discount codes for the following hotels in Leeds so <a href="mailto:">drop us a message</a> if
+        you're
```
Comment (benadamstyles) [inline src/travel-accommodation.html]:
```suggestion
        you&rsquo;re
```

**Review by benadamstyles — CHANGES_REQUESTED** [review]

File: src/reset.less (hunk 1 above)
Comment (Danielle-P) [reply src/reset.less]: Changed font size for all a tags within p tags instead

**Review by benadamstyles — APPROVED** [review]

**Pull request merged**